ansible/roles/kafka-logging-setup/tasks/main.yml

```diff
@@ -1,15 +1,31 @@
 ---
-- name: Deploy Kafka cluster
+- name: Label Kafka nodes
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'node.yml.j2') }}"
+  with_items: "{{ kafka_nodes | default ([]) }}"
```

Comment:
Not for this pr, but I think for the future, we will migrate all node labeling into a separate role so we don't have a chicken before the egg issue with local-storage needing kafka labels but the labeling being in the kafka playbook.  Lets save that for another pr though.